mkdocs.yml

```diff
@@ -125,34 +133,32 @@ plugins:
             heading_level: 4
   - social
   - distilabel/components-gallery:
-      add_after_page: Learn
+      add_after_page: How-to guides
 
 nav:
   - Distilabel: "index.md"
   - Getting started:
-    - Installation: "sections/installation.md"
-    - How-to-Guide: "sections/how_to_guide.md"
-  - Learn:
-    - "sections/learn/index.md"
-    - Tutorial:
-      - "sections/learn/tutorial/index.md"
-      - Step:
-        - "sections/learn/tutorial/step/index.md"
-        - GeneratorStep: "sections/learn/tutorial/step/generator_step.md"
-        - GlobalStep: "sections/learn/tutorial/step/global_step.md"
-      - Task:
-        - "sections/learn/tutorial/task/index.md"
-        - GeneratorTask: "sections/learn/tutorial/task/generator_task.md"
-      - LLM: "sections/learn/tutorial/llm/index.md"
-      - Pipeline: "sections/learn/tutorial/pipeline/index.md"
-      - CLI: "sections/learn/tutorial/cli/index.md"
+    - Installation: "sections/getting_started/installation.md"
+    - Quickstart: "sections/getting_started/quickstart.md"
+    - FAQ: "sections/getting_started/faq.md"
+  - How-to guides:
+    - Basic:
+      - Define Steps as nodes in a Pipeline:
+        - "sections/how_to_guides/basic/step/index.md"
+        - GeneratorStep: "sections/how_to_guides/basic/step/generator_step.md"
+        - GlobalStep: "sections/how_to_guides/basic/step/global_step.md"
+      - Define Tasks as Steps that rely on LLMs:
+        - "sections/how_to_guides/basic/task/index.md"
+        - GeneratorTask: "sections/how_to_guides/basic/task/generator_task.md"
+      - Define LLMs as local models or remote APIs: "sections/how_to_guides/basic/llm/index.md"
+      - Execute Steps and Tasks in a Pipeline: "sections/how_to_guides/basic/pipeline/index.md"
```

Comment:
Same here as mentioned above, I'm not really convinced with the naming, but don't have a proposal / suggestion either :/

Reply:
@burtenshaw might have some other ideas.